appveyor.yml

```diff
@@ -73,9 +76,7 @@ after_test:
   - python clcache.py -s
 
   # Upload coverage data to codecov.io for analysis
-  - bash --version
-  - curl --version
-  - curl -s https://codecov.io/bash | bash -
+  - codecov
```

Comment:
Turns out that bash and curl are indeed available - but using the Python uploader which was suggested by the codecov.io founder (who actually listens to the web chat facility!) seems kinda nice, so let's use that.